gm4_mountaineering/data/gm4_mountaineering/functions/ski_effects/steep_downhill_ski_speed/check_steep_downhill_ski_speed.mcfunction

```diff
@@ -0,0 +1,13 @@
+# Checks that the player still travels steep downhill (>=7 blocks, or >=5 blocks with poles)
+# @s = player with skiing tag and having fallen >=7 blocks, or >=5 blocks with poles
+# at @s
+# run from skis_equipped
+
+execute if entity @s[scores={gm4_mountaineering_ski_steep_downhill=1..},predicate=!gm4_mountaineering:holding_poles] unless score @s gm4_mountaineering_vy matches ..-7 run scoreboard players remove @s gm4_mountaineering_ski_steep_downhill 1
+execute if entity @s[scores={gm4_mountaineering_ski_steep_downhill=1..},predicate=gm4_mountaineering:holding_poles] unless score @s gm4_mountaineering_vy matches ..-5 run scoreboard players remove @s gm4_mountaineering_ski_steep_downhill 1
```

Comment:
Why do you do score checks two different ways? the `unless score ...` subcommand can be combined with the `if entity ...` subcommand